labgrid/pytestplugin/fixtures.py

```diff
@@ -74,7 +80,8 @@ def env(request, record_testsuite_property):
         try:
             target = env.get_target(target_name)
         except UserError as e:
-            pytest.exit(e)
+            pytest.exit(str(e))
```

Comment:
This change seems to be unrelated to typing? Or does it fix a bug? If so, best to put the bug-fixes first, in one or more separate commits.

Reply:
Actually, this fixes that the wrong type is passed to the `pytest.exit()` function: [pytest.exit](https://github.com/pytest-dev/pytest/blob/main/src/_pytest/outcomes.py#L107).